domain/builders/atrium-domain-builders-common/src/main/kotlin/ch/tutteli/atrium/domain/builders/creating/anyAssertionsBuilders.kt

```diff
@@ -20,7 +20,7 @@ import kotlin.reflect.KClass
  * In detail, it implements [AnyAssertions] by delegating to [anyAssertions]
  * which in turn delegates to the implementation via [loadSingleService].
  */
-object AnyAssertionsBuilder : AnyAssertions {
+object AnyAssertionsBuilder : AnyAssertions by anyAssertions {
```

Comment:
please revert

Reply:
Oh, that was an experiment. Should not have been commited. Done.